scripts/import_standard_ebooks.py

```diff
@@ -0,0 +1,158 @@
+import json
+from typing import Dict, List, Optional
```

Comment:
Let's please use the new syntax...
```suggestion
from __future__ import annotations

import json
```
Then use `dict`, `list`, and [xyz | None]

Reply:
Done, except not using square brackets for the third case.